specs/catalog-apicurio.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/apicurio/config/resolve/artifact/global/id/cache/zilla.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+catalogs:
+  catalog0:
+    type: apicurio
+    options:
+      url: http://localhost:8080
+      group-id: groupId
+bindings:
+  net0:
+    type: test
+    kind: server
+    options:
+      catalog:
+        catalog0:
+          - id: 1
+      assertions:
+        catalog:
+          catalog0:
+            - schema: "asyncapi: 3.0.0\ninfo:\n  title: Zilla MQTT Proxy\n  version: 1.0.0\n  license:\n    name: Aklivity Community License\nservers:\n  plain:\n    host: mqtt://localhost:7183\n    protocol: mqtt\ndefaultContentType: application/json"
+            - schema: "asyncapi: 3.0.0\ninfo:\n  title: Zilla MQTT Proxy\n  version: 1.0.0\n  license:\n    name: Aklivity Community License\nservers:\n  plain:\n    host: mqtt://localhost:7183\n    protocol: mqtt\ndefaultContentType: application/json"
```

Comment:
Can we use yaml syntax like `|` or `>` here to put the asyncapi spec on multiple lines below for improved readability but still parse into the same overall string with spaces and newlines as needed?